src/main/java/com/meilisearch/sdk/http/URLBuilder.java

```diff
@@ -52,6 +52,10 @@ public URLBuilder addParameter(String parameter, String[] value) {
         return this;
     }
 
+    public String addQuery(String query) {
```

Comment:
When I suggested in the meeting to create this method, I was expecting this:

```java
public URLBuilder addQuery(String query) {
  this.addSeparator();
  this.params.append(query);

  return this;
}
```

Reply:
Yes, I was wondering if it was better to send the object back, and then we could do the following:
`urlb.addQuery(param.ToQuery).getURL();`

Reply:
I think it is better to change to the `URLBuilder` because `addQuery` gives me the feeling that I could keep appending stuff, like: `addSubroute(...).addSeparator()...`.

Reply:
Indeed